api/users/forms.py

```diff
@@ -29,11 +29,9 @@ class InitConfigForm(forms.Form):
     )
 
     def create_admin(self):
```

Comment:
Can `create_admin` be run multiple times? If so, the `create_initial_superuser` function name isn't quite right since it could be used for multiple superusers.

Reply:
> Can `create_admin` be run multiple times?

It's not supposed to be as the view using the form gets closed off after the user is created.